.github/workflows/main.yml

```diff
@@ -87,9 +87,48 @@ jobs:
       - name: Integration
         run: |
           export SINON_CHROME_BIN=$(which google-chrome-stable)
+
+          # test-coverage runs test-headless in Chrome
+          npm run test-coverage  -- --chrome "$SINON_CHROME_BIN" --allow-chrome-as-root
+
           npm run test-headless -- --chrome $SINON_CHROME_BIN --allow-chrome-as-root
           npm run test-webworker -- --chrome $SINON_CHROME_BIN --allow-chrome-as-root
           npm run test-esm-bundle
+      - name: Upload coverage report
+        run: bash <(curl -s https://codecov.io/bash) -F unit -s coverage/lcov.info
+
+  saucelabs-test:
+    if: ${{ github.ref == 'refs/heads/master' }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16]
```

Comment:
Why is this using a matrix with a single version?

Reply:
Because lazy. I'll tidy it up